dpx/src/dpx_pdfobj.rs

```diff
@@ -231,12 +233,12 @@ impl Default for xref_entry {
     }
 }
 
-#[derive(Copy, Clone)]
+use indexmap::IndexMap;
+
+#[derive(Clone)]
```

Comment:
This clone does not deeply clone it's elements, and is very prone to use-after-free for its elements